iam-policies/terraform/README.md

```diff
@@ -9,7 +9,7 @@ This Terraform module deploys additional Elastio IAM managed policies that you c
 ```tf
 module "elastio_policies" {
   source  = "terraform.cloudsmith.io/public/elastio-iam-policies/aws"
-  version = "0.33.1"
+  version = "0.33.2"
```

Comment:
I guess, we can use the `latest` version in the README and examples

Reply:
I'd keep the exact version to make it easier for the users. I'm pretty sure they'll copy the `latest` version if we don't specify an exact one here